_account-security/security/encryption/encryption-general.md

```diff
@@ -0,0 +1,113 @@
+---
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Data Encryption
+permalink: /account-security/data-encryption
+summary: "Stitch offers secure options for making connections to all data sources and destinations, giving you the power to secure your data as you see fit."
+
+input: false
+layout: general
+feedback: false
+
+# -------------------------- #
+#        Introduction        #
+# -------------------------- #
+
+intro: |
+  {% capture security-faq %}
+  **Looking for general security info?** Check out the [Security FAQ]({{ link.security.faq | prepend: site.baseurl }}).
+  {% endcapture %}
+
+  {% include note.html type="single-line" content=security-faq %}
+
+  Our most important job here at Stitch is to keep your data safe. To do that, Stitch always encrypts data in transit and at rest within the Stitch environment.
+
+  {{ page.summary }}
+
+  In this guide, we'll cover Stitch's supported connection options and provide links to additional resources:
+
+  {% for section in page.sections %}
+  - [{{ section.title }}](#{{ section.anchor }})
+  {% endfor %}
+
+# -------------------------- #
+#           Content          #
+# -------------------------- #
+
+
+ssh-tunnels:
+  - name: "Self-hosted"
+    guide: "ssh-generic"
+    description: "If your database is hosted on your server and not in the cloud, it's considered a 'self-hosted' database. This is applicable to both integrations and destinations."
+
+  - name: "Amazon"
+    guide: "ssh-amazon"
+    description: "Stitch currently supports connecting Amazon RDS and Amazon Redshift (destination only) databases."
+
+  - name: "Microsoft Azure"
+    guide: "ssh-microsoft-azure"
+    description: "Stitch currently supports connecting Microsoft Azure SQL Server (as an integration) and Azure SQL Data Warehouse (as a destination). Other Microsoft Azure offerings aren't currently supported."
+
+sections:
+  - title: "SSL connections"
+    anchor: "ssl-connections"
+    content: |
+      [Secure Sockets Layer](https://www.verisign.com/en_US/website-presence/online/ssl-certificates/index.xhtml){:target="new"}, or SSL, is a standard security technology used to establish encrypted communication between a web server and a browser. SSL ensures that communication to and from Stitch remains private and secure.
+
+    subsections:
+      - title: "Stitch application access"
```

Comment:
I think this section is potentially confusing since it talks about SSL in a completely different context. Doesn't the general security FAQ section already cover this topic?

Reply:
Change 'login credentials' to something like 'communictation'